waiter/src/waiter/token.clj

```diff
@@ -621,7 +627,13 @@
                                                         kv-store token
                                                         :error-on-missing false
                                                         :include-deleted include-deleted)]
-                                 (every? #(% token-parameters) parameter-filter-predicates))))
+                                 (and (every? #(% token-parameters) parameter-filter-predicates)
+                                      (or (nil? include-run-as-requester?)
+                                          (boolean-xnor include-run-as-requester?
+                                                        (sd/run-as-requester? token-parameters)))
```

Comment:
is it better to write
(or (nil? include-run-as-requester?)
      (= (boolean include-run-as-requester?) (sd/run-as-requester? token-parameters)))
?

Reply:
Yes, I also prefer `=` over `boolean-xnor` :grimacing:

Reply:
Done.